stream_alert_cli/runner.py

```diff
@@ -0,0 +1,344 @@
+                        'https://github.com/airbnb/streamalert/issues')
+    LOGGER_CLI.info(cli_load_message)
+
+    if options.command == 'lambda':
+        lambda_runner(options)
+
+    elif options.command == 'terraform':
+        terraform_runner(options)
+
+
+def lambda_runner(options):
+    """Handle all Lambda CLI operations"""
+    if options.subcommand == 'deploy':
+        deploy(options)
+
+    elif options.subcommand == 'rollback':
+        rollback()
+        targets = ['module.stream_alert_{}'.format(x)
+                   for x in CONFIG['clusters'].keys()]
+        tf_runner(targets=targets)
+
+    elif options.subcommand == 'test':
+        stream_alert_test(options)
+
+
+def terraform_check():
+    """Verify that Terraform is configured correctly"""
+    prereqs_message = ('Terraform not found! Please install and add to'
+                       'your $PATH:\n'
+                       '$ export PATH=$PATH:/usr/local/terraform/bin')
+    run_command(['terraform', 'version'],
+                error_message=prereqs_message,
+                quiet=True)
+
+
+def terraform_runner(options):
+    """Handle all Terraform CLI operations"""
+    # verify terraform is installed
+    terraform_check()
+    # use a named tuple to match the 'processor' attribute in the argparse options
+    deploy_opts = namedtuple('DeployOptions', ['processor'])
+
+    # plan/apply our streamalert infrastructure
+    if options.subcommand == 'build':
+        # --target is for terraforming a specific streamalert module
+        if options.target:
+            target = options.target
+            targets = ['module.{}_{}'.format(target, cluster)
+                       for cluster in CONFIG['clusters'].keys()]
+            tf_runner(targets=targets)
+        else:
+            tf_runner()
+
+    # generate terraform files
+    elif options.subcommand == 'generate':
+        generate_tf_files()
+
+    # initialize streamalert infrastructure from a blank state
+    elif options.subcommand == 'init':
+        LOGGER_CLI.info('Initializing StreamAlert')
+        LOGGER_CLI.info('Generating Cluster Files')
+        generate_tf_files()
+
+        # build init infrastructure
+        LOGGER_CLI.info('Building Initial Infrastructure')
+        init_targets = [
+            'aws_s3_bucket.lambda_source',
+            'aws_s3_bucket.integration_testing',
+            'aws_s3_bucket.terraform_remote_state',
+            'aws_kms_key.stream_alert_secrets',
+            'aws_kms_alias.stream_alert_secrets'
+        ]
+        tf_runner(targets=init_targets, refresh_state=False)
+
+        LOGGER_CLI.info('Deploying Lambda Functions')
+        # setup remote state
+        refresh_tf_state()
+        # deploy both lambda functions
+        deploy(deploy_opts('all'))
+        # create all remainder infrastructure
+        LOGGER_CLI.info('Building Remainder Infrastructure')
+        tf_runner()
+        # refresh config to get modified variables
+        # refresh_config()
```

Comment:
Can this be deleted? As a general rule, there shouldn't be commented code without a reason.